src/etc/inc/captiveportal.inc

```diff
@@ -347,15 +347,10 @@ EOD;
 		@unlink("{$g['varetc_path']}/captiveportal-{$cpzone}-error.html");
 		@unlink("{$g['varetc_path']}/captiveportal-{$cpzone}-logout.html");
 
-		captiveportal_radius_stop_all();
+		captiveportal_disconnect_all(10); // NAS-Request
 
 		captiveportal_filterdns_configure();
 
-		/* send Accounting-Off to server */
-		if (!platform_booting()) {
-			captiveportal_send_server_accounting(true);
```

Comment:
Seems then it would be better to test for the proper condition to send or not send, rather than remove it. At the very least I'd move that to a separate bug/PR since it is not directly related to this change.